src/cm/profile.rs

```diff
@@ -46,6 +48,10 @@ impl Profile {
                         cmd_count += 1;
                         result.cmd_list.list.items.push(value.to_string());
                     }
+                    "shells" => {
+                        shell_count += 1;
+                        result.shell_list.list.items.push(value.to_string());
+                    }
```

Comment:
Any changes to the default configuration file require migrating old configs. See `migraiton.rs` file.

Reply:
Provided a migration now, thanks for explaining it.